src/Cake.Common/Tools/GitReleaseNotes/IssueTracker.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Cake.Common.Tools.GitReleaseNotes
+{
+    /// <summary>
+    /// The IssueTracker to be used.
+    /// </summary>
+    public enum IssueTracker
```

Comment:
@gep13 as this basically becomes a global available enum we should probably prefix if with GitReleaseNotes.